devicetypes/smartthings/smartsense-moisture-sensor.src/README.md

```diff
@@ -0,0 +1,36 @@
+# Smartsense Moisture Sensor
+
+
+
+Works with: 
+
+* [Samsung SmartThings Moisture Sensor](https://shop.smartthings.com/#!/products/samsung-smartthings-water-leak-sensor)
+
+## Table of contents
+
+* [Capabilities](#capabilities)
+* [Health]($health)
+* [Battery]($battery)
+
+## Capabilities
+
+* **Configuration** - _configure()_ command called when device is installed or device preferences updated
+* **Battery** - defines device uses a battery
+* **Refresh** - _refresh()_ command for status updates
+* **Temperature Measurement** - defines device measures current temperature
+* **Water Sensor** - can detect presence of water (dry or wet)
+* **Health Check** - indicates ability to get device health notifications
+
+## Device Health
+
+A Category C2 moisture sensor that has 120min check-in interval
```

Comment:
This value `7200` might be misleading; perhaps we can put how often it periodically reports back and how we decide to 2x it.